### PR TITLE
11 create point class

### PR DESCRIPTION
**Pull request by piotrv1001**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -103,20 +103,48 @@ def __del__(self):
...
+
 class Dragable:
     @staticmethod
     def is_dragable():
         return True
 
 
-class Point(Dragable):
-    y_pos = 500
+class Point(Entity, Dragable):
```
Comment (ArturOle) [inline app.py:138]: Draggable will no longer be useful for us. We can just delete it for now

File: app.py [hunk 2]
```diff
@@ -103,20 +103,48 @@ def __del__(self):
...
     radius = 10
     _green = (0, 255, 0)
     _red = (255, 0, 0)
 
-    def __init__(self, pos, val):
-        self.x_pos = pos
+    def __init__(self, x_pos, val):
+        if val:
+            super().__init__(x_pos, self._green, x_pos + self.radius)
```
Comment (ArturOle) [inline app.py:145]: Why the half-length is x_pos + radius? Isn't it just the distance from the point's center?

File: app.py [hunk 3]
```diff
@@ -103,20 +103,48 @@ def __del__(self):
...
     @staticmethod
     def is_dragable():
         return True
 
 
-class Point(Dragable):
-    y_pos = 500
+class Point(Entity, Dragable):
     radius = 10
     _green = (0, 255, 0)
```
Comment (ArturOle) [inline app.py:140]: Shouldn't the colors and the radius be in the Entity class?

Comment (piotrv1001) [reply app.py]: Colors, yes. Radius seems specific for a point(circle).

**Pull request merged**